bin/build-languages.js

```diff
@@ -1,48 +1,24 @@
 const crypto = require( 'crypto' );
 const fs = require( 'fs' );
-const https = require( 'https' );
+const { writeFile } = require( 'fs' ).promises;
 const path = require( 'path' );
-const readline = require( 'readline' );
 const languages = require( '@automattic/languages' );
 const parse = require( 'gettext-parser' ).po.parse;
-const glob = require( 'glob' );
 const _ = require( 'lodash' );
 const mkdirp = require( 'mkdirp' );
+const fetch = require( 'node-fetch' );
 
 const LANGUAGES_BASE_URL = 'https://widgets.wp.com/languages/calypso';
 const LANGUAGES_REVISIONS_FILENAME = 'lang-revisions.json';
-const CALYPSO_STRINGS = './calypso-strings.pot';
-const CHUNKS_MAP_PATTERN = './chunks-map.*.json';
+const CALYPSO_STRINGS = './public/calypso-strings.pot';
+const CHUNKS_MAP_PATTERN = './public/chunks-map.json';
```

Comment:
I like the idea of keeping the root directory clean and I suppose there's no concern for these to be publicly accessible.